.github/actions/ledger-nano/action.yml

```diff
@@ -0,0 +1,20 @@
+name: 'ledger-nano-setup'
+description: 'Setup a simulated ledger nano instance'
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout ledger shimmer app files
+      uses: actions/checkout@v3
+      with:
+        repository: iotaledger/ledger-shimmer-app
+        path: ledger-shimmer-app
```

Comment:
ledger-shimmer-app is obsolete. It was all merged into ledger-iota-app and for the `build.sh` you need to specify the variant with `-v shimmer`

Reply:
Is that all that you think needs to be done?

Reply:
Still getting `DeviceNotFound` in the test.